pkg/monitoring/metrics/virt-controller/vmstats_collector_test.go

```diff
@@ -368,4 +370,67 @@ var _ = Describe("VM Stats Collector", func() {
 			Expect(crs[2].Labels).To(Equal([]string{"testvm", "test-ns", "cpu", "sockets"}))
 		})
 	})
+
+	Context("PVC allocated size metric collection", func() {
+		BeforeEach(func() {
+			persistentVolumeClaimInformer, _ = testutils.NewFakeInformerFor(&k8sv1.PersistentVolumeClaim{})
+		})
+
+		It("should collect PVC size metrics correctly", func() {
```

Comment:
I think unit tests should cover several scenarios: empty volumeMode is just an example

Reply:
added more tests